ts/test/session/unit/utils/i18n/getRawMessage_test.ts

```diff
@@ -2,46 +2,49 @@
 // @ts-nocheck - TODO: add generic type to setupI18n to fix this
 
 import { expect } from 'chai';
-import { initI18n, testDictionary } from './util';
-import { resetLocaleAndTranslationDict } from '../../../../../util/i18n/shared';
+import { initI18n } from './util';
 
 describe('getRawMessage', () => {
-  let i18n;
-  beforeEach(() => {
-    i18n = initI18n(testDictionary);
-  });
-
-  afterEach(() => {
-    resetLocaleAndTranslationDict();
-  });
-
   it('returns the raw message for a token', () => {
-    const rawMessage = i18n.getRawMessage('greeting', { name: 'Alice' });
-    expect(rawMessage).to.equal('Hello, {name}!');
+    const rawMessage = initI18n().getRawMessage('en', 'adminPromoteDescription', { name: 'Alice' });
```

Comment:
Could you not just do this once and then reference it? Same in `getMessage_test.ts`?

Reply:
It's testing and that saves me a beforeEach/afterEach. I think that's fine?

Reply:
Assuming there is no performance impact since I guess they do the same thing then it's fine?

Reply:
There is no cleanup if you don't call afterEach though given a test fails while the others are running in parallel?

Reply:
I like it like this more :)

Reply:
https://youtu.be/4fndeDfaWCg?t=28